bin/lamassu-blacklist

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+const os = require('os')
+const readline = require('readline')
+
+const coinUtils = require('../lib/coin-utils')
+const db = require('../lib/db')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+const sql = `select regexp_replace(crypto_code, '$', ' ') as code,regexp_replace(address, '^', ' ') as address from blacklist`
+const insertSql = `insert into blacklist(crypto_code, address) values ($1, $2)`
+
+db.query(sql)
+  .then(many => {
+    console.log('')
+    console.log('Here is your current list of prohibited addresses:')
```

Comment:
```suggestion
    console.log('Here is your current list of prohibited addresses:\n')
```

Then you don't need all the empty ``console.log``s.

Reply:
I've used os.EOL.